daml-lf/engine/src/main/scala/com/digitalasset/daml/lf/engine/Engine.scala

```diff
@@ -98,7 +103,10 @@ final class Engine {
   ): Result[Transaction.Transaction] = {
     for {
       commands <- Result.sequence(ImmArray(nodes).map(translateNode(_commandTranslation)))
-      result <- interpret(submitters, commands, ledgerEffectiveTime)
+      lfVers <- CommandsVersion(_compiledPackages, commands.map(_._2.templateId))
+      // TODO(FM) is it wrong to assume that we're validating here? presumably not,
```

Comment:
Remove this TODO

Reply:
i.e. drop or convert to a comment helping the reader with definite information

Reply:
@oggy- can you comment here?